README.md

```diff
@@ -1,6 +1,59 @@
 # Express Wolox Logger
 
-ExpressJS logger that wraps [pino](https://github.com/pinojs/pino) with additional features like unique requestId and a requestMiddleware.
+ExpressJS logger that wraps [pino](https://github.com/pinojs/pino) with additional features like middlewares for unique request ids and automatic logging of request beginnings and endings.
+
+## Basic Usage
+```
+const { logger } = require('express-wolox-logger');
+
+logger.info('hello world');
+logger.error('something bad happened');
+```
+This will output:
+```
+[2019-06-14 17:35:13.772 +0000] INFO  (17439 on my-pc.local): hello world
+[2019-06-14 17:35:13.772 +0000] ERROR (17439 on my-pc.local): something bad happened
+```
+
+## Advanced Usage
+### Logs for request beginning and end
+We provide an ExpressJs middleware that automatically logs when a request starts and ends. Simply import it and use it like any other middleware:
+```
+const { expressMiddleware } = require('express-wolox-logger');
+
+app.use(expressMiddleware);
+```
+This in conjunction with the basic logs will output:
+```
+[2019-06-14 17:35:13.770 +0000] INFO  (17439 on my-pc.local): Started GET /logger/test with params: {}, query: {}, body: {}.
+[2019-06-14 17:35:13.772 +0000] INFO  (17439 on my-pc.local): hello world
+[2019-06-14 17:35:13.772 +0000] ERROR (17439 on my-pc.local): something bad happened
+[2019-06-14 17:35:13.781 +0000] INFO  (17439 on my-pc.local): Ended GET /logger/test with status: 200 in 10 ms
+```
+
+### Request Ids
+We also provide an ExpressJs middleware that appends a `request id` to all logs made for a single request. This is useful for better tracking logs when there are several requests going on concurrently. Again, simply import it and use it like any other middleware.
+```
+const { expressRequestIdMiddleware } = require('express-wolox-logger');
+
+app.use(expressRequestIdMiddleware);
+```
+This, in conjunction with the basic logs will output:
+```
+[2019-06-14 17:35:13.772 +0000] INFO  (17439 on my-pc.local): [a2936029-9bd4-402d-ba43-a4873f228274] hello world
+[2019-06-14 17:35:13.772 +0000] ERROR (17439 on my-pc.local): [a2936029-9bd4-402d-ba43-a4873f228274] something bad happened
+```
+####Forwarding the request Id
+As a bonus, the previously mentioned request Id is taken from the `x-request-id` header if supplied, which lets said `request id` be transferred across services. You can do this by importing the `getRequestId` function and supplying it to the header when making requests.
+```
+const request = require('axios'),
+ { getRequestId } = require('express-wolox-logger');
+
+axios.get(URL, { headers: { 'xrequest-id': getRequestId() } });
```

Comment:
This should be `request.get(...` or rename const to `axios`

Reply:
Also typo in the header name